perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -217,6 +282,7 @@ public RichMember call(ApiCaller ac, Deserializer params) throws PerunException
 		public RichMember call(ApiCaller ac, Deserializer params) throws PerunException {
 			params.stateChangingCheck();
 			String password = params.readString("password");
+			String login = params.readString("login");
```

Comment:
I think this way of reading makes the API param required, while it should actually depend on the namespace. You should read it like a "validityTo" param later in the code.

Also how this (in the call hierarchy) handles the case, when to be sponsored user already have login in namespace and we provided new login from input?